jsk_pcl_ros/src/pointcloud_screenpoint_nodelet.cpp

```diff
@@ -132,10 +132,14 @@ void jsk_pcl_ros::PointcloudScreenpoint::onInit()
 
 bool jsk_pcl_ros::PointcloudScreenpoint::checkpoint (pcl::PointCloud< pcl::PointXYZ > &in_pts, int x, int y,
                                                      float &resx, float &resy, float &resz)  {
-  if ((x < 0) || (y < 0) || (x >= in_pts.width) || (y >= in_pts.height)) return false;
+  if ((x < 0) || (y < 0) || (x >= in_pts.width) || (y >= in_pts.height)) {
+    JSK_ROS_WARN_STREAM("Requested point is out of image size.  point: ("
+                        << x << "," << y << ")" << "  size: (" << in_pts.width << "," << in_pts.height << ")");
```

Comment:
How do you switch to use stream and printf? I'd like to know general criteria.
Maybe I'll do same mistake in the next time, and I'd like to reduce chance to modify PR.